tests/drivers/gpio/gpio_arduino/testcase.yaml

```diff
@@ -0,0 +1,21 @@
+common:
+  tags:
+    - drivers
+    - gpio
+  depends_on:
+    - gpio
+    - arduino_gpio
+  harness: ztest
+  harness_config:
+    fixture: arduino_gpio
+
+tests:
+  drivers.gpio.arduino:
+    filter: not dt_compat_enabled("arduino-header-r3")
```

Comment:
do not understand this filter, you want to include everything but those with `arduino-header-r3` or the opposite, given that the test is about arduino

Reply:
sorry this is a debug code. I need change back